### PR TITLE
Preview iteration

### PR DESCRIPTION
**Pull request by the-daniel-rothig**
### Context

https://manage-courses-prototype.herokuapp.com/history/preview-course

### Changes proposed in this pull request

Necessary updates

### Guidance to review

I was going to write something helpful but a meeting is starting.

### CODE REVIEW

**Comment by defong** [comment]
Should I be seeing markdown?



snc-markdown

**Comment by dankmitchell** [comment]
@defong for now, yes. Theres a card to strip this out in the future

**Comment by defong** [comment]
@dankmitchell its not markdown

**Comment by the-daniel-rothig** [comment]
@defong `== headers ==` are in the data which will not be carried across... a "drop database" will fix that for you locally ;). I updated the financial support copy to match the prototype, thanks for spotting

**Comment by defong** [comment]
See https://github.com/DFE-Digital/manage-courses-ui/pull/146

for additional comments

File: shared/Views/Shared/Components/CourseDetails/Default.cshtml [hunk 1]
```diff
@@ -158,40 +182,13 @@
                 </div>
             }
 
-            @if (Model.PreviewMode || Model.HasSection(CourseDetailsSections.EntryRequirementsQualifications))
+            @if (Model.HasSection(CourseDetailsSections.InterviewProcess))
             {
                 <div class="govuk-!-margin-bottom-8">
-                    <h2 class="govuk-heading-m" id="section-entry">
-                        Requirements
+                    <h2 class="govuk-heading-m" id="section-about">
```
Comment (fofr) [inline shared/Views/Shared/Components/CourseDetails/Default.cshtml]: Why is the about section pointing to the interview process?

**Pull request merged**